app/models/solid_cache/entry.rb

```diff
@@ -7,7 +7,7 @@ class Entry < Record
     # 2. We avoid the overhead of building queries and active record objects
     class << self
       def write(key, value)
-        upsert_all_no_query_cache([ { key: key, value: value } ])
```

Comment:
I don't think value omission is available in Ruby 3.0, so we'll need to keep specifying them (here and below)

Reply:
Apologies, I didn't intend to change those lines. I'll have a stern discussion with my linter settings.